platform/golden/provisioners/roles/override_cuda_driver_parameters/files/dropin-profile-all-for-cuda.cfg

```diff
@@ -0,0 +1 @@
+options nvidia NVreg_EnableGpuFirmware=0 NVreg_EnableStreamMemOPs=1 NVreg_NvLinkDisable=1 NVreg_RestrictProfilingToAdminUsers=1 NVreg_DynamicPowerManagement=0 NVreg_EnablePCIERelaxedOrderingMode=1 NVreg_EnableResizableBar=1 NVreg_DmaRemapPeerMmio=0 NVreg_EnableGpuFirmwareLogs=0
```

Comment:
@rstarmer your machine is using this. I am not sure that is is activated, and I am heading to the gym. If you run into trouble, please let me know!